raster/src/main/scala/geotrellis/raster/merge/SinglebandTileMergeMethods.scala

```diff
@@ -27,65 +27,49 @@ import spire.syntax.cfor._
   * single-band [[Tile]]s.
   */
 trait SinglebandTileMergeMethods extends TileMergeMethods[Tile] {
-  /** Merges this tile with another tile.
```

Comment:
Feels like we should add a delegate with `baseCol = 0, baseRow = 0` to be a little more conservative with the interface.

Reply:
Is [this](https://github.com/jpolchlo/geotrellis/blob/3b799a6b3774243a3608c4ae70a56af74db33f57/raster/src/main/scala/geotrellis/raster/merge/TileMergeMethods.scala#L38) not sufficient?

Reply:
Huh, missed that in the diff somehow, that's totally sufficient.